widgetssdk/src/main/java/com/glia/widgets/chat/adapter/CustomCardAdapter.java

```diff
@@ -44,7 +44,7 @@ public abstract class CustomCardAdapter {
      *         return null;
      *     }
      * }<pre/>
-     * @param message the chat message with metadata
+     * @param message The chat message with metadata.
```

Comment:
It sounds like "the" is not needed here and in other similar places because it's about any chat message rather than some exact one.